recipe/meta.yaml

```diff
@@ -17,17 +17,17 @@ build:
 requirements:
   host:
     - python >=3.10
+    - setuptools >=61.0.0
+    - setuptools-scm >=7.0
     - pip
   run:
     - python >=3.10
     - xarray >=2024.06.0
-    - kerchunk >=0.2.5
-    - h5netcdf
-    - pydantic
```

Comment:
These are listed as test dependencies. Maybe they were meant to be optional dependencies here? Looking at the package metadata it is now clear. See https://github.com/zarr-developers/VirtualiZarr/blob/v1.1.0/pyproject.toml#L34-L50

Reply:
@TomNicholas do you mind taking a look at this before merging?

Reply:
fwiw this is accurate, but I agree that some of these may be better as optional dependencies. I will open a PR in VirtualiZarr for Tom and others to review.